samples/react/ElmishCalc/src/State.fs

```diff
@@ -0,0 +1,77 @@
+module Calc.State
+
+open Calc.Types
```

Comment:
I think it makes it clear that I want the `Types` concerning the calc no other types. Keeping the same convesion for small and  large (nested) components. Other than that it has no significant value to choose one or the other so I will keep it this way :)